### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by Mottie**
* Remove "Mozilla Format" header and move icon to the right of the import/export buttons as mentioned in our discussion in Discord.
* Make editable help modal styles match the manage help modal style. Only the informational popups have extra padding around the content.

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -865,7 +865,7 @@
...
+    "message": "Choose the type of sort to apply to the installed entries from within the sort dropdown. The default setting applies an ascending sort (A to Z) to the entry titles. Sorts within the \"Title Descending\" group will apply a descending sort (Z to A) to the title.\nThere are other presets that will allow sorting the entries by multiple criteria. Think of this like sorting a table with multiple columns and each category in a select (between the plus signs) represents a column, or group.\nFor example, if the setting is \"Enabled + Title\", the entries would sort so that all the enabled entries are sorted to the top of the list, then an entry title ascending sort (A to Z) is applied to both the enabled and disabled entries separately.",
```
Comment (tophf) [inline _locales/en/messages.json]: Changing stuff in messages.json will delete the existing translation in every language. I don't think one word warrants this amount of hassle.

Comment (Mottie) [reply _locales/en/messages.json]: Ok, I removed that commit.

**Comment by tophf** [comment]
1. The margin above `Options` heading now seems too big. I think it should match the one between `Enabled` and the buttons.
2. What about reducing the vertical margin between the buttons so it's equal to the horizontal inter-button margin?
3. I think `div` should be used just like for the first row of buttons, and it should have `display:flex; align-items:center`
4. Might be reasonable to add tooltips to the Import/Export buttons. For example, "Mozilla Format" (`styleMozillaFormatHeading`), if it's not confusing.

**Comment by Mottie** [comment]
Done; but, I didn't include the `display: flex` because it was removing the spacing between the import and export buttons.

**Comment by tophf** [comment]
Spacing can be added explicitly for both button rows. Without align:center the icon is not centered vertically.

**Comment by narcolepticinsomniac** [comment]
What's the objective here? Why are we losing the header?

**Comment by tophf** [comment]
1. It's wasting space which we need to add a block for user exclusions.
2. It's wasting space which could be used to display more linting issues on smaller displays
3. It splits the button block into two for no good reason

**Comment by narcolepticinsomniac** [comment]
>It's wasting space which we need to add a block for user exclusions.

OK then. Figured you were making room, but wasn't sure what for.

>It splits the button block into two for no good reason

Meh, I think that visually the buttons looked slightly nicer split up, if we had the room to spare. Grouping them all together is a little asymmetrical. NBD if we need the room though.

**Comment by Mottie** [comment]
I included the `align-items: center;`. Was there something else that needed adjustment?

**Pull request closed without merging**